research/src/pages/radio-button/radio-button.proposal.mdx

```diff
@@ -0,0 +1,185 @@
+
+#### CSS Parts
+
+| Slot Name           | Description                                                   |
+| ------------------- | ------------------------------------------------------------- |
+| `label`             | The control's label (and implicit value)                      |
+| `checked-indicator` | Indicates the radio button is in a checked or unchecked state |
+
+## Radio Button Group Design
+
+This section relates to a `<oui-radio-button-group>`. It can host multiple `<oui-radio-button>` elements.
+
+### API
+
+#### Properties
+
+| Attribute Name | Type     | Default Value | Description                                                                                      |
+| -------------- | -------- | ------------- | ------------------------------------------------------------------------------------------------ |
+| `name`         | `string` | `null`        | Represents the name of the control, to be used upon form submission .                            |
+| `value`        | `string` | `null`        | The value of the selected choice.                                                                |
+| `autofocus`    | `bool`   | `false`       | Get focus by default. Defaults to the first option, unless explicitly defined among the options. |
+| `disabled`     | `bool`   | `false`       | Prevents the user from interacting with any of the choices. Defaults to false.                   |
+| `form`         | `string` | `null`        | Associates the element with a form in the document whose `id` is this value.                     |
+| `readonly`     | `bool`   | `false`       | Makes it unavailable for user interaction, but the selected choice should still be submitted.    |
+| `required`     | `bool`   | `false`       | Makes it required to have a selected choice.                                                     |
+
+#### Events
+
+...
+
+### Anatomy
+
+<RadioButtonGroupAnatomy />
+
+#### DOM Structure (Implicit Grouping)
+
+The following structure represents a group with the `choices` slot filled with radio buttons (their internal structures are omitted for brevity).
+
+```html
+<oui-radio-button-group name="group-name">
+  <slot name="label">
+    <label part="label"></label>
+  </slot>
+  <slot name="required-indicator">
+    <div part="required-indicator"></div>
+  </slot>
+  <slot name="choices">
+    <div part="choices">
+      <oui-radio-button />
+      <oui-radio-button />
+      <oui-radio-button />
+      ...
+    </div>
+  </slot>
+</oui-radio-button-group>
+```
+
+#### DOM Structure (Explicit Grouping)
+
+The following structure, in contrast, represents a group with an empty `choices` slot. The choices are placed outside the group, providing layout flexibility. Each choice has a `group` attribute with value equal to the radio button group's name, making them explicitly bound as choices of that group.
+
+There is also the possibility of having both implicit and explicit choices.
```

Comment:
As noted elsewhere, so long as we can produce the correct structure for AT, I love this.